sn_cli/src/subcommands/files/mod.rs

```diff
@@ -423,11 +454,11 @@ pub(crate) async fn upload_files(
     println!("Uploading {chunks_to_upload_len} chunks",);
     let now = Instant::now();
     let upload_result = match files_upload.upload_chunks(chunks_to_upload).await {
-        Ok(()) => {Ok(())}
+        Ok(()) => { Ok(()) }
         Err(ClientError::Transfers(WalletError::Transfer(TransfersError::NotEnoughBalance(
-            available,
-            required,
-        )))) => {
+                                                             available,
```

Comment:
i'd wager `cargo fmt` is complaining about this. 

We use that for standardised indentation.

Reply:
ah yes, fmt did not like that

Reply:
I use cargo fmt and the CTRL + shift  + F my ide almost like a muscle reflex. 
I will keep this in mind.